### PR TITLE
Heap 34916

### PR DESCRIPTION
**Pull request by murphpdx**
_A summary of your pull request, including the what change you're making and why._
- We've added session_id to our payload, we would like customers to be able to pass this parameter in to track user sessions. We will eventually use this with a mobile extension. 
- I've updated the segment library for tracking purposes on our end. 

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/browser-destinations/src/destinations/heap/constants.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const HEAP_SEGMENT_LIBRARY_NAME = 'destinations-actions'
+export const HEAP_SEGMENT_LIBRARY_NAME = 'browser-destination'
```
Comment (nick-Ag) [inline packages/browser-destinations/src/destinations/heap/constants.ts:1]: Nit: Maybe rename the two `HEAP_SEGMENT_LIBRARY_NAME` constants to be distinct from each other

**Review by zikaari — CHANGES_REQUESTED** [review]
Looks good. Please remove console.log statements

**Review by jinapark202 — APPROVED** [review]

**Comment by jinapark202** [comment]
@murphpdx I think you may need to update your snapshot tests as they are failing

**Comment by brennan** [comment]
@murphpdx Quick reminder to please fix the unit tests in this integration - they we can review, approve, and merge! Once fixed, we'll deploy as part of our next release (Nov. 16) after the current change freeze.

**Comment by dlasky** [comment]
@murphpdx just checking back in here, are we still looking to merge these changes? Looks like nothing has changed on the testing front?

**Comment by murphpdx** [comment]
> @murphpdx just checking back in here, are we still looking to merge these changes? Looks like nothing has changed on the testing front?

Looks like I lost track of this one. I updated it, I think it's ready to go now?

**Review by pooyaj — APPROVED** [review]

**Comment by murphpdx** [comment]
Hi @pooyaj @zikaari @jinapark202, I don't have permission to merge this. Can someone merge for me? Thank you.

**Comment by NolanChan** [comment]
Hi @murphpdx , I just ran the CI and it seems like some Heap tests failed?

**Pull request merged**